src/protocol/logpoint.ts

```diff
@@ -66,7 +66,7 @@ client.Analysis.addAnalysisResultListener(({ analysisId, results }) => {
     results.forEach(
       async ({ key, value: { time, pauseId, location, values, data, frameworkListeners } }) => {
         const pause = new Pause(ThreadFront.sessionId!);
-        pause.instantiate(pauseId);
+        pause.instantiate(pauseId, key, time, true);
```

Comment:
Can this either have a comment like `pause.instantiate(pauseId, key, time, /* hasFrames */ true)` or pass an object literal to improve readability?

Reply:
Done